packages/lambda/src/lib/store/block-index.js

```diff
@@ -64,11 +64,10 @@ export class BlockIndexClaimFetcher extends DynamoTable {
           const [, bucket, ...rest] = carpath.split('/')
           const key = rest.join('/')
           const part = bucketKeyToPartCID(key)
-          const origin = encodeURIComponent('r2://auto/carpark-prod-0')
 
           // derive location URL(s) from the key
           location = [
-            ...(part ? [new URL(`https://w3s.link/ipfs/${part}?format=raw&origin=${origin}`)] : []),
+            ...(part ? [new URL(`https://carpark.w3s.link/${part}/${part}.car`)] : []),
```

Comment:
Just now noticed this, with this we will not have region, and not clear bucket name. This would for instance be problematic for roundabout to fully move to location claims. Can we just have both?